device/common/include/traccc/finding/device/apply_interaction.hpp

```diff
@@ -8,29 +8,31 @@
 #pragma once
 
 // Project include(s).
+#include "detray/navigation/navigator.hpp"
+#include "detray/propagator/actors/pointwise_material_interactor.hpp"
 #include "traccc/definitions/qualifiers.hpp"
 #include "traccc/finding/finding_config.hpp"
+#include "traccc/utils/particle.hpp"
 
 namespace traccc::device {
+template <typename detector_t>
+struct apply_interaction_payload {
+    typename detector_t::view_type det_data;
+    const int n_params;
+    bound_track_parameters_collection_types::view params_view;
+    vecmem::data::vector_view<const unsigned int> params_liveness_view;
```

Comment:
Some Doxygen documentation should be put on the variables, and the struct as well. Since now that documentation is not on the main function anymore.

Reply:
Done